<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBagno</class>
 <widget class="QWidget" name="WBagno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>60</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Anticamera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="armadiettoLavBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Armadietto Lavandino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="armadioBagnoBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Armadio Bagno</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
